### ISSUE TITLE
[EN][QUEST][Burning Steppes]Strategic Cuts

### ISSUE THREAD

**Issue by terranu1** [Atlantiss/BugTracker#8289]
Worgmistress Othana cannot be stabed with the quest weapon .

**Comment by Olszi37** [comment]
http://www.wowhead.com/quest=28246/strategic-cuts

True. Stabbing her and nothing

**Comment by ryotucker** [comment]
This is still an issue

**Comment by Zygmanta** [comment]
Right-clicking the buff that keeps you allied with blackrock will remove you from alliance and if you don't move while near her, you can kill her normally and it counts.

**Comment by ryotucker** [comment]
OK, thank you!

**Comment by MercuricUma** [comment]
The quest item provided worked only on the guy on the tower (Voodoist Timan). For the others, remove your disguise and just kill them. Still counts for the quest

**Comment by MikanTsu** [comment]
BTW, don't remove the disguise buff in the middle of her dome as her wolfs nearly killed me.
Plan the attack first!
